### DISCUSSION TITLE
Project Downlaod fails from Cloud

### DISCUSSION THREAD

**Discussion by MonikaShah** [opengisch/QField#3727] [Q&A]
![Screenshot from 2022-12-07 14-02-47](https://user-images.githubusercontent.com/4188149/206150526-b99d109a-ed80-4b05-bc91-70d71d8e2bd7.png)

**Comment by suricactus** [comment]
Posting you questions twice will not speed up answering you. Duplicate of https://github.com/opengisch/QField/discussions/3724

**Comment by MonikaShah** [comment]
I am sorry for repeating , but actually it needs an urgent resolving.

**Comment by mbernasocchi** [comment]
Dear @MonikaShah,
We are sorry you are encountering an issue, but as @suricactus mentioned, duplicating issues only slows down the process, if you need urgent support, please check the support options at https://qfield.org/support

Cheers Marco

**Comment by MonikaShah** [comment]
Thank you for your response.
I am writing to support team
